packages/react-devtools-shared/src/devtools/views/ErrorBoundary/ReportNewIssue.js

```diff
@@ -13,9 +13,18 @@ import {searchGitHubIssuesURL} from './githubAPI';
 import styles from './shared.css';
 
 function encodeURIWrapper(string: string): string {
-  return encodeURI(string).replace(/#/g, '%23');
+  return encodeURI(string).replace(/\n/g, '%0A');
```

Comment:
Is this a browser quirk? In chrome 90 both `encodeURI` and `encodeURIComponent` already replace `\n`.
![Screenshot from 2021-05-07 10-43-04](https://user-images.githubusercontent.com/12292047/117423510-1a013380-af21-11eb-948d-e1717594d0b0.png)

Reply:
Unclear to me. It didn't replace this string properly before I explicitly escaped it.